GOOD-I-DEER/nodes/webcam-object-detection-node/webcam-html.js

```diff
@@ -223,28 +224,29 @@ module.exports.code = (config) => {
                 return result;
             }
 
-           function iou(box1,box2) {
-                return intersection(box1,box2)/union(box1,box2);
+           function iou(box1, box2) {
+                return intersection(box1, box2) / union(box1, box2);
             }
 
-           function union(box1,box2) {
-                const [box1_x1,box1_y1,box1_x2,box1_y2] = box1;
-                const [box2_x1,box2_y1,box2_x2,box2_y2] = box2;
-                const box1_area = (box1_x2-box1_x1)*(box1_y2-box1_y1)
-                const box2_area = (box2_x2-box2_x1)*(box2_y2-box2_y1)
-                return box1_area + box2_area - intersection(box1,box2)
+           function union(box1, box2) {
+                const [box1_x1, box1_y1, box1_x2, box1_y2] = box1;
+                const [box2_x1, box2_y1, box2_x2, box2_y2] = box2;
+                const box1_area = (box1_x2 - box1_x1) * (box1_y2 - box1_y1);
+                const box2_area = (box2_x2 - box2_x1) * (box2_y2 - box2_y1);
+                return box1_area + box2_area - intersection(box1,box2);
```

Comment:
Auto-sorting is god... 👀

Reply:
In fact, because the code is in backticks, it does not automatically align.... so I sorted it manually 🤣